importers/research-media-csv-importer.php

```diff
@@ -0,0 +1,249 @@
+<?php
+/**
+ * Imports thumbnails for researchers from WordPress Exports
+ */
+namespace UCF\MainSiteUtilities\Importers {
+	class CSVThumbnailImporter {
+		private
+			$csv_data,
+			$base_url,
+			$source_meta_key,
```

Comment:
`$source_meta_key` doesn't appear to be in use anywhere in this class and could be removed